scripts/launch/wallet.sh

```diff
@@ -9,7 +9,10 @@ if [[ "$SSC_ALGO" != "" ]]; then
     ssc_algo=" --ssc-algo $SSC_ALGO "
 fi
 
+system_start=100500 # random value, not used, but mandatory
+
 $(find_binary cardano-wallet) $(peer_config 0) $(logs smartwallet$i.log) \
                               --db-path "run/wallet-db" --rebuild-db \
                               --flat-distr "(3, 100000)" $ssc_algo "$@" \
-                              --system-start 100500  # random value, not used, but mandatory
+                              --system-start $system_start \
+                              repl
```

Comment:
Why `repl`? I am quite surprised that repl works. And I have never used it, because I find it more convenient to pass commands as arguments.
Anyway, I think `wallet.sh` should support both modes, just like `cardano-wallet` does (so if you need repl, you can use `wallet.sh repl`).
You can see how I modified it in #1215.